src/lib/operators/alias_operator.cpp

```diff
@@ -0,0 +1,68 @@
+#include "alias_operator.hpp"
+
+#include <sstream>
+
+#include "boost/algorithm/string/join.hpp"
+#include "storage/table.hpp"
+
+using namespace std::string_literals;
+
+namespace opossum {
+
```

Comment:
Maybe we should squeeze aliasing into the abstract operator?

Reply:
In my opinion, we would not gain much if we make the abstract operator even bigger but keep the alias node class. So either keep both of them or none.